.github/workflows/nodejs.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         run: npm run security
 
       - name: Check commit message
-        uses: wagoid/commitlint-github-action@v1
+        uses: wagoid/commitlint-github-action@v4
```

Comment:
/cc @snitin315 Make sense to update everywhere

Reply:
I will update 👍